apis/python/src/tiledbsoma/io/outgest.py

```diff
@@ -145,8 +158,14 @@ def _extract_X_key(
     X = measurement.X[X_layer_name]
     tp = X.context.threadpool
 
+    if dask:
+        return load_daskarray(
```

Comment:
how do you pass timestamp info to the Dask loader?  I.e., if the user is opening the Experiment in the past, does it get the same result as the non-Dask variant?   (this would make a good unit test!)

Reply:
Actually reading the code, this is really a question for the `load_daskarray` function.  I'll duplicate it there.